translations.json

```diff
@@ -1217,9 +1237,9 @@
     "^TM": "™",
     "_a": "ₐ",
     "_e": "ₑ",
+    "_h": "ₕ",
     "_i": "ᵢ",
     "_j": "ⱼ",
-    "_h": "ₕ",
```

Comment:
This is the only modification that's not only addition. I reorder the letters a bit so it follows the order of alphabet to for easy lookup.